CRM/Event/BAO/Event.php

```diff
@@ -891,7 +887,9 @@ public static function &getCompleteInfo(
         $info['location'] = $address;
         $info['url'] = CRM_Utils_System::url('civicrm/event/info', 'reset=1&id=' . $dao->event_id, TRUE, NULL, FALSE);
 
-        if ($enable_cart) {
+        // @todo Move to eventcart extension
+        // check if we're in shopping cart mode for events
```

Comment:
So we   could merge the change to this file now - ie the comments are OK & the changes don't rely on the extension